### DISCUSSION TITLE
Golang client?

### DISCUSSION THREAD

**Discussion by selfagency** [pocketbase/pocketbase#4726] [Q&A] [answered]
I know Pocketbase is written in Go, but is there a Golang client or SDK?

**Comment by ganigeorgiev** [comment, marked as answer]
There is no official Go SDK (see also https://github.com/pocketbase/pocketbase/discussions/995).

Other users shared the below packages but I haven't tested them:

- https://github.com/r--w/pocketbase
- https://github.com/pluja/pocketbase (_a fork of the above with support for the most recent versions at least based on the readme_)
